config/initializers/100-sidekiq.rb

```diff
@@ -51,16 +51,7 @@ def print_banner
   end
 end
 
-# Sidekiq#logger= applies patches to whichever logger we pass it.
-# Therefore something like Sidekiq.logger = Rails.logger will break
```

Comment:
@davidtaylorhq I had a look at Sidekiq's code and it doesn't mess with the logger if `Sidekiq.logger == Rails.logger`: 

https://github.com/mperham/sidekiq/blob/f58cd04cd374a3a4e72741afdd25e1251846eae1/lib/sidekiq/rails.rb#L45

Reply:
O I found the problem, let me see if I can fix it.